pkg/wallet/wallet.go

```diff
@@ -18,21 +17,20 @@ import (
 	"github.com/massalabs/thyra-plugin-wallet/api/server/models"
 	"golang.org/x/crypto/pbkdf2"
 	"golang.org/x/exp/slices"
+	"gopkg.in/yaml.v2"
 	"lukechampine.com/blake3"
 )
 
 // TODO: rename to 'account'
 
 const (
 	SecretKeyLength           = 32
-	PBKDF2NbRound             = 600000
+	PBKDF2NbRound             = 60_0000
```

Comment:
I guess there is an issue here
```suggestion
	PBKDF2NbRound             = 600_000
```

Reply:
good catch